src/main/steps/respondent/check-your-answers/content.ts

```diff
@@ -248,7 +248,7 @@ const languages = {
 
 export const generateContent: TranslationFn = content => {
   const translations = languages[content.language](content);
-  const applicant2Url = content.isApplicant2 ? urls.RESPONDENT : '';
+  const applicant2Url = urls.RESPONDENT;
```

Comment:
wouldnt this break where its being used, as itll be underfined

Reply:
when would it be undefined?

Reply:
https://github.com/hmcts/nfdiv-frontend/pull/1557#discussion_r772215279

Reply:
yep, youre right, ignore me, i was thinking if it was checking if its app2